js/build.js

```diff
@@ -242,7 +242,7 @@ Fliplet.Widget.instance('login-ds', function(data) {
           _this.removeClass('loading');
           _this.find('.btn-label').removeClass('hidden');
           _this.find('.loader').removeClass('show');
-          _this.parents('.form-btns').find('.login-error').html('Your email or password don\'t match. Please try again.').removeClass('hidden');
+          _this.find('.login-error').html('Your email or password don\'t match. Please try again.').removeClass('hidden');
```

Comment:
What gets shown to the user if we just use the server's error response? A `.catch()` being triggered doesn't always mean the email or password don't match. The server should be managing the scenarios (or show the JavaScript error if the error is local to the client.)

```suggestion
          _this.find('.login-error').html(Fliplet.parseError(error)).removeClass('hidden');
```

Reply:
Tested. Everything works. Fixed